AppCenterData/AppCenterDataTests/MSDBDocumentStoreTests.m

```diff
@@ -382,8 +388,10 @@ - (void)testDeleteAppDocumentForNonExistentDocument {
 - (void)testDeleteExistingAppDocument {
 
   // If
-  MSDocumentWrapper *documentWrapper = [MSDocumentUtils documentWrapperFromData:[self jsonFixture:@"validTestDocument"]
+  MSDocumentWrapper *documentWrapper = [MSDocumentUtils documentWrapperFromData:[self jsonFixture:@"validTestAppDocument"]
                                                                    documentType:[MSDictionaryDocument class]
+                                                                      partition:@"user-123"
```

Comment:
You don't need a custom prefixed user partition it's only used for the user table in the storage. These calls are more likely to use kMSDataUserDocumentsPartition.

Reply:
I was wrong the returned metadata from the document wrapper should contain the user-account.